Assets/Editor/AmplitudePostProcessor.cs

```diff
@@ -0,0 +1,23 @@
+﻿using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEditor.iOS.Xcode;
+
+public static class AnalyticsPostProcessor {
```

Comment:
Keep the class name consistent? `AmplitudePostProcessor`?

Reply:
Yep sorry, it's a mistake ^^